### PR TITLE
ITPL-5637 Edited postgres to support oracle

### PR DESCRIPTION
**Pull request by CExBBartolome**

### CODE REVIEW

File: lib/create-common-client.js [hunk 1]
```diff
@@ -125,6 +127,32 @@ module.exports = function (config) {
...
+        commonClient.dbDriver = require('oracle');
+
+        commonClient.queries.getCurrentVersion = 'SELECT * FROM (SELECT "version" FROM "' + config.schemaTable + '" ORDER BY "version" DESC)' + ' WHERE ROWNUM<2';
+        commonClient.queries.checkTable = "SELECT * FROM ALL_OBJECTS WHERE object_type = 'TABLE' AND object_name = '" + config.schemaTable + "'";
+        commonClient.queries.makeTable = 'CREATE TABLE "' + config.schemaTable + '" (version NUMBER(38,0) PRIMARY KEY); INSERT INTO "' + config.schemaTable + '" ("version") VALUES (0);';
+
+        var connection = {
+                    host: config.host,
+                    user: config.username,
+                    password: config.password,
```
Comment (literallynotjesus) [inline lib/create-common-client.js:141]: Missing port btw.

Comment (CExBBartolome) [reply lib/create-common-client.js:141]: good catch, thanks @CExJBoudreau !

Comment (CExBBartolome) [reply lib/create-common-client.js:141]: will patch

Comment (CExBBartolome) [reply lib/create-common-client.js:141]: done v4.1.1

**Pull request merged**